app/javascript/components/upload/StepTabHeader.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+/** renders the wizard step header for a given step */
+export default function StepTitle({ step, index, currentStep, setCurrentStep, serverState, formState }) {
+  let stepFiles = []
+  if (formState && formState.files) {
+    stepFiles = formState.files.filter(step.fileFilter)
+  }
+  return <li className={step.name === currentStep.name ? 'active' : ''} onClick={() => setCurrentStep(step)}>
```

Comment:
Less dense for easier reading:

```suggestion
  const className = step.name === currentStep.name ? 'active' : ''
  return <li className={className} onClick={() => setCurrentStep(step)}>
```